src/rpc/rawtransaction.cpp

```diff
@@ -202,6 +202,8 @@ UniValue getrawtransaction(const JSONRPCRequest& request)
             "     }\n"
             "     ,...\n"
             "  ],\n"
+            "  \"extraPayloadSize\" : n    (numeric) Size of DIP2 extra payload. Only present if it's a special TX\n"
+            "  \"extraPayload\" : \"01234\"  (string) Hex encoded DIP2 extra payload data. Only present if it's a special TX\n"
```

Comment:
Just noticed this. The hex encoded stuff are usually `\"hex\"` in rpc help and not `\"01234\"`.

Reply:
fixed